tests/baselines/reference/objectRestNegative.errors.txt

```diff
@@ -34,8 +33,6 @@ tests/cases/conformance/types/rest/objectRestNegative.ts(17,9): error TS2701: Th
                                     ~

```

Comment:
SHould probably remove this `function generic<T ...` example since the new test covers exactly the same case.

Reply:
Yeah, but no harm in keeping it.